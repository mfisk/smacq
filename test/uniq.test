#!/bin/sh

bin/smacqq -m -O <<EOF 2>/dev/null
print ts, srcip, dstip from (uniq srcip, dstip from pcapfile($TEST/0325@1112-snort.pcap))
EOF
