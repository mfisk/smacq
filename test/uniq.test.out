954011609.360668	192.168.0.54	192.168.0.210
954011728.020237	192.168.0.54	192.168.0.200
954011731.922436	192.168.0.200	192.168.0.54
